### ISSUE TITLE
How are asynchronous actions executed?

### ISSUE THREAD

**Issue by shyam573** [TengHu/ActionWeaver#157]
The current implementation does not seem to support asynchronous function calls. Is there a way to execute them directly?

**Comment by TengHu** [comment]
Correct, the async function calling is not supported yet, is it a blocking feature for you ?

**Comment by shyam573** [comment]
I am currently managing the actions by using asyncio calls and changing my functions to sync. It would be nice if I need not write another function to handle this or keep my set of functions as it is
